### PR TITLE
Adds basic info about shard.override.yml file to `shards` command page

### PR DESCRIPTION
**Pull request by crimson-knight**
The title says it all, I basically took the man page info and just added an example. This should make it a lot easier for users to find.

### CODE REVIEW

File: docs/man/shards/README.md [hunk 1]
```diff
@@ -126,3 +126,22 @@ shards version [<path>]
 ```
 
 Prints the version of the shard.
+
+## Fixing Dependency Version Conflicts
+
+A `shard.override.yml` file allows overriding the source and restriction of dependencies. An alternative location can be configured with the env var SHARDS_OVERRIDE.
```
Comment (straight-shoota) [inline docs/man/shards/README.md:132]:
```suggestion
A `shard.override.yml` file allows overriding the source and restriction of dependencies. An alternative location can be configured with the env var `SHARDS_OVERRIDE`.
```

File: docs/man/shards/README.md [hunk 2]
```diff
@@ -126,3 +126,22 @@ shards version [<path>]
...
+
+The file contains a YAML document with a single dependencies key. It has the same semantics as in shard.yml. Dependency configuration takes precedence over the configuration in shard.yml or any dependency’s shard.yml.
```
Comment (straight-shoota) [inline docs/man/shards/README.md:134]:
```suggestion
The file contains a YAML document with a single `dependencies` key. It has the same semantics as in `shard.yml`. Dependency configuration takes precedence over the configuration in `shard.yml` or any dependency’s `shard.yml`.
```

**Review by straight-shoota — COMMENTED** [review]
Some markup would be nice.

**Comment by crimson-knight** [comment]
@straight-shoota sure thing! Certainly makes some of the description pop and easier to read

**Review by straight-shoota — APPROVED** [review]

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**